src/Fixers/NoCommentFixer.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Codestyle\Fixers;
+
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\Fixer\Whitespace\NoExtraBlankLinesFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+use PhpCsFixerCustomFixers\TokenRemover;
+use SplFileInfo;
+
+final class NoCommentFixer implements FixerInterface
```

Comment:
Will it remove all comments in the code? E.g.
```php
// some helpful info about this var
protected int $xyz = 1;
```

Reply:
Yes. That's why it's intended only for one run for basic Laravel installation. It should not be used during development later and that's why it's not in `CommonRules`.